phpcs.xml.dist

```diff
@@ -1,4 +1,11 @@
 <?xml version="1.0"?>
-<ruleset name="/dev/null">
+<ruleset name="WordPress Coding Standards">
+	<config name="installed_paths" value="vendor/wp-coding-standards/wpcs" />
 
+	<rule ref="WordPress" />
+
+	<arg name="extensions" value="php" />
+
+	<file>.</file>
+	<exclude-pattern>/vendor/</exclude-pattern>
 </ruleset>
```

Comment:
Can we use a derivative of https://github.com/pantheon-systems/wp-saml-auth/blob/master/phpcs.xml.dist ? It has a few more dodads that are useful.

Reply:
Yeah, sound good. I can do this tomorrow (Tuesday).